src/main/scala/is/hail/utils/TextTableReader.scala

```diff
@@ -162,7 +163,8 @@ object TextTableReader {
     nPartitions: Int = hc.sc.defaultMinPartitions,
     quote: java.lang.Character = null,
     keyNames: Array[String] = Array.empty[String],
-    skipBlankLines: Boolean = false): Table = {
+    skipBlankLines: Boolean = false,
+    useCols: Array[String] = Array.empty[String]): Table = {
```

Comment:
It might be nicer to use an `Option` or null here -- null/None being the default behavior of "use all columns"

Reply:
also let's make it a `Set[String]` -- order probably shouldn't matter here.